app.json

```diff
@@ -8,6 +8,7 @@
   "env": {
     "BUILDPACK_URL": "https://github.com/RocketChat/heroku-buildpack-meteor.git",
     "ROOT_URL": {
+			"value": "https://YOUR-APP-NAME.herokuapp.com"
```

Comment:
We had this before, and more people got confused and left the default value there.